msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -361,6 +361,9 @@ If the app requests for permissions change, the users are prompted to upgrade an
 
 This version string must follow the [semver](http://semver.org/) standard (MAJOR.MINOR.PATCH).
 
+> [!NOTE]
```

Comment:
@Rick-Kirkham, 

Shouldn't the GA file also be updated. Please confirm. We will then merge the PR.